index.html

```diff
@@ -2174,11 +2174,11 @@ <h3>Refreshing</h3>
         </p>
         <p>
 Including the refresh reference inside the <a>verifiable presentation</a>
```

Comment:
This section has conflicts with a previous PR#408. PR#408 should be resolved first before this section on "Refreshing" can be properly reviewed.